_episodes_rmd/06-vector-open-shapefile-in-r.Rmd

```diff
@@ -103,8 +103,15 @@ Our `aoi_boundary_HARV` is a polygon object. The 18 levels shown below
 our output list the possible categories of the geometry type. 
 Now let's check what CRS this file data is in:
 
+Remembering from an earlier lesson that CRS information uses a mathematical model  to connect data to the Earth’s surface.
+
+When we read in the shapefile into an R object, the metadata included the CRS – in this case is 32618
+We can get more information about the CRS with the 
+st_crs() function, the CRS code number, and method to get information on the CRS in the proj4string.   
+
+
 ```{r}
-st_crs(aoi_boundary_HARV)
+st_crs(32618)$proj4string
```

Comment:
What happens if you do `st_crs(aoi_boundary_HARV)$proj4string` here instead - are the results the same? I'm not sure I know what 32618 is :/ and I think it might be good to see how to check the crs of the `aoi_boundary_HARV` object using the `st_crs()` function. That being said, I don't know what 32618 is so this could be totally fine. I know you mentioned that this function has changed recently. Open to suggestions from both @cidfreitag and @sjgraves

Reply:
Good suggestion, thanks! I find that using  st_crs(aoi_boundary_HARV)$proj4string  results in 
[1] "+proj=utm +zone=18 +datum=WGS84 +units=m +no_defs "

which is the output in the lesson